include/ck/tensor_operation/gpu/device/impl/device_grouped_gemm_xdl_splitk_cshuffle.hpp

```diff
@@ -365,13 +365,17 @@ struct DeviceGroupedGemmXdlSplitKCShuffle : public DeviceGroupedGemmSplitK<ALayo
         index_t skipped_group_count_;
 
         std::vector<GemmTransKernelArg> gemm_kernel_args_;
+        GemmTransKernelArg* gemm_kernel_host_args_;
```

Comment:
We can have it here even: 
```suggestion
        void* gemm_kernel_host_args_;
```